src/ca11y/index.js

```diff
@@ -4,32 +4,93 @@ import zeroPad  from './lib/zeroPad'
 import keys     from './lib/keys'
```

Comment:
Yeah, looks like it's no longer used by anything. I think you should remove it, and the package...?